web-client/public/js/setup-handlers.js

```diff
@@ -92,31 +93,29 @@ export const setupHandlers = grnState => {
         $("#exportAsSvg").attr("download", editedName);
     };
 
-    const setGraphLayout = layout => {
-        const different = grnState.graphLayout !== layout;
-        grnState.graphLayout = layout;
-        if (different) {
-            if (grnState.graphLayout === FORCE_GRAPH) {
-                updaters.setNodesToForceGraph();
-            } else if (grnState.graphLayout === GRID_LAYOUT) {
-                updaters.setNodesToGrid();
-            }
-        }
-
-        updateApp(grnState);
-    };
-
     // Grid and Force Graph Layout
-    $(GRID_LAYOUT_BUTTON).click(() => {
-        setGraphLayout(grnState.graphLayout === FORCE_GRAPH ? GRID_LAYOUT : FORCE_GRAPH);
+    $(FORCE_GRAPH_BUTTON).click(() => {
+        updaters.setNodesToForceGraph();
+        grnState.graphLayout = FORCE_GRAPH;
+        updateApp(grnState);
     });
 
     $(FORCE_GRAPH_CLASS).click(() => {
-        setGraphLayout(FORCE_GRAPH);
+        updaters.setNodesToForceGraph();
+        grnState.graphLayout = FORCE_GRAPH;
+        updateApp(grnState);
+    });
+
+    $(GRID_LAYOUT_BUTTON).click(() => {
+        updaters.setNodesToGrid();
+        grnState.graphLayout = GRID_LAYOUT;
+        updateApp(grnState);
     });
 
     $(GRID_LAYOUT_CLASS).click(() =>  {
-        setGraphLayout(GRID_LAYOUT);
+        updaters.setNodesToGrid();
+        grnState.graphLayout = GRID_LAYOUT;
+        updateApp(grnState);
```

Comment:
Looks like these click handlers can be consolidated—what’s here right now is duplication of two functions into four. Does putting both selectors in the same jQuery call work? For example:
```javascript
$(FORCE_GRAPH_BUTTON, FORCE_GRAPH_CLASS).click( // .....etc.
```
This way, the functions for setting nodes to force graph vs. grid aren’t duplicated.

(if a comma-separated list doesn’t work, I’m sure we can come up with a way to make both selectors work together)

Reply:
Multiple selectors will work, but the format we have our constants in does not work. This selector requires ` $("#forceGraph, #forceGraphButton").click(() => { //...` where multiple selectors are contained inside the same string. This: `$(FORCE_GRAPH_BUTTON, FORCE_GRAPH_CLASS).click( // ...` will not work, because it is passing multiple separate strings. Is there a way to clean this up?